gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinClusterUtils.java

```diff
@@ -254,4 +257,13 @@ public static FileSystem buildFileSystem(Config config, Configuration conf)
         .get(URI.create(config.getString(ConfigurationKeys.FS_URI_KEY)), conf)
         : FileSystem.get(conf);
   }
+
+  public static FileSystem buildNewInstanceFileSystem(Config config, Configuration conf) throws IOException {
```

Comment:
`ddm.work.assistance.Help` already has a number of varieties of `loadFileSystem`.  this may not duplicate any of the ones there right now, but should we anyway define it over there and look into the possibility of merging this w/ one of those other impls?

Reply:
This isn't being used by Temporal so unsure if I should be associating the Helper class for temporal inside of this class, this also means that any existing yarn app used for streaming will need to add a dependency on gobblin-temporal module

Reply:
ok, let's not do that then.